packages/experiments/src/components/FloatingSuggestions/Suggestions/SuggestionsControl.scss

```diff
@@ -0,0 +1,82 @@
+@import '~office-ui-fabric-react/src/common/common';
+
+.root {
```

Comment:
Since we intend to rewrite this component after the UnifiedPicker is merged in and consumable, I'm gonna say we ought to just write the new suggestions as styled and move forward with this as it is.